src/controllers/canadian.controller.ts

```diff
@@ -111,6 +113,40 @@ export class CanadianController extends SessionController<CanadianBlueLinkyConfi
   /* eslint-disable @typescript-eslint/no-explicit-any */
   private async request(endpoint, body: any, headers: any = {}): Promise<any | null> {
     logger.debug(`[${endpoint}] ${JSON.stringify(headers)} ${JSON.stringify(body)}`);
+    const [major,,] = process.versions.node.split('.').map(Number);
+    if (major >= 21) {
+      process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```

Comment:
I saw we are setting `NODE_TLS_REJECT_UNAUTHORIZED` to disable TLS validation, I thought we only need to override via `SSL_OP_LEGACY_SERVER_CONNECT`?